docs/cli/kops_completion.md

```diff
@@ -0,0 +1,38 @@
+## kops completion
+
+Outputs kops shell completion for the given shell (bash)
+
+### Synopsis
+
+
+Outputs kops shell completion for the given shell (bash)
+
+```
+kops completion
+```
+
+### Options
+
+```
+      --shell string   target shell (bash).
```

Comment:
I know k8s has examples, how do we look at getting those in the docs? Outside the scope of this PR, probably, let me know if u want me to open an issue.